packages/idempotency/tests/unit/persistence/DynamoDbPersistenceLayer.test.ts

```diff
@@ -13,11 +13,12 @@ import type { DynamoDBPersistenceOptions } from '../../../src/types';
 import { IdempotencyRecordStatus } from '../../../src';
 import {
   DynamoDBClient,
-  DynamoDBServiceException,
+  //DynamoDBServiceException,
```

Comment:
How come this is commented out?

Reply:
I have removed `DynamoDBServiceException` in favour of `ConditionalCheckFailedException` since the error been thrown itself is an instance of `ConditionalCheckFailedException`

Reply:
Makes sense! Let's remove the line, we don't want commented out code in the codebase